src/uas/UAS.cc

```diff
@@ -2986,9 +2969,9 @@ float UAS::addZeroMeanNoise(float truth_meas, float noise_var)
     
     //TODO add bias term that changes randomly to simulate accelerometer and gyro bias the exf should handle these
     //as well
-    float noise = z0 * (noise_var*noise_var); //calculate normally distributed variable with mu = 0, std = var^2  
+    float noise = z0 * sqrtf(noise_var); //calculate normally distributed variable with mu = 0, std = sqrt(var)

```

Comment:
This is indeed a mistake on my part.  It should be sqrt(variance) not variance*variance.  

This change should also be reflected in tools/RandomNumberVerification testing harness.  The typo made it through the test because variance was misdefined in both the matlab script and the c++ funtion.  

My mistake